SportsHub/SportsHub.Application/Authentication/Models/Options/PasswordHashingOptions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SportsHub.AppService.Authentication.Models.Options
+{
+    public sealed class HashingOptions
```

Comment:
Name of file and class are different - is this intentional?

Reply:
It wasn't intentional, the class is now renamed.